spec/namespaces/_core.yaml

```diff
@@ -5067,11 +5067,19 @@ components:
     mtermvectors::path.index:
       in: path
       name: index
-      description: Name of the index that contains the documents.
+      description: The name of the index that contains the document.
       required: true
       schema:
         $ref: '../schemas/_common.yaml#/components/schemas/IndexName'
       style: simple
+    mtermvectors::path.id:
+      in: path
+      name: id
+      description: The unique identifier of the document.
+      required: true
+      schema:
+        $ref: '../schemas/_common.yaml#/components/schemas/Id'
+      style: simple
```

Comment:
You've defined a new path parameter here which is unused

Reply:
@Xtansia Excellent catch, thats left over from previous PR that was abandoned - this has been removed now